submissions/Roka20012/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,112 @@
+const delayForAnswer = 1000;
+const delay = 500;
```

Comment:
Use conventional uppercase for constants and more meaningful var name for `delay`.

Reply:
Fix it thanks)